dockerfiles/docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
       - proxito
       - storage
     volumes:
-      - ${PWD}/dockerfiles/nginx:/etc/nginx/conf.d
+      - ${PWD}/dockerfiles/nginx:/etc/nginx/templates
```

Comment:
Really good idea to start using envsubst :+1: This is a good step towards convergence between development docker images and production. Ideally, the same Docker images can be used in multiple deployments and after having isolated variables as environment variables, we can reduce reliance on volumes and build configurations straight into the images.

Reply:
Note that this is incompatible with current `readthedocs-corporate` setup.